### PR TITLE
Doc 10614  txns stub

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+[abstract]
+{description}
+
+
+Distributed ACID Transactions are implemented in the Java SDK, and simple to integrate into your Kotlin client code.
+
+
+== Java Transactions from the Scala SDK
+
+Just create a `com.couchbase.client.java.Cluster` in your Kotlin app with the xref:java-sdk:howtos:managing-connections.adoc[normal Java calls] `(Cluster.connect(...))`, and then do transactions on it as normal, e.g.:
```
Comment (dnault) [inline modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc]: NIT: Is saying "just" or "simply" a disservice to our users? Whether something is "simple" or not depends on what you know going in :-)

If we've done a good job explaining something, do we also need to say it's simple?

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+[source,java]
+----
+javaCluster.transactions().run(ctx => { /* your transaction logic here */ })
+----
+
+So in your app you have both a `com.couchbase.client.java.Cluster` and a `com.couchbase.client.kotlin.Cluster`, which are independent connections.
+
+Note, you will need to include the Couchbase Java SDK in your project.
```
Comment (dnault) [inline modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc]: Is this burying the lede?

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+include::project-docs:partial$attributes.adoc[]
+
+[abstract]
+{description}
+
+
+Distributed ACID Transactions are implemented in the Java SDK, and simple to integrate into your Kotlin client code.
+
+
+== Java Transactions from the Scala SDK
```
Comment (dnault) [inline modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc]: s/Scala/Kotlin ?

**Review by dnault — CHANGES_REQUESTED** [review]
Thanks for adding this, Richard!

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc]: Fair point.

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc]: Yeah, let's change this.

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc (hunk 3 above)
Comment (RichardSmedley) [reply modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc]: :facepalm:

**Review by dnault — APPROVED** [review]
LGTM!

**Pull request merged**